### DISCUSSION TITLE
Nice Project!

### DISCUSSION THREAD

**Discussion by blastentwice** [TCA166/CK3-history-extractor#4] [General]
Just want to drop in to say that you made an amazing tool for ck3. It's also inspiring for me too as I'm reworking my own project. Keep up the good work!

**Comment by TCA166** [comment]
Thank you for your kind words! I do indeed try and hold my projects to the highest standard. Chronicles are probably next on the docket, but I have to figure out if squeezing in an LLM into the executable is feasible. I wish you good luck with your project. Though if I were you, I would ditch Python for a more structured language. While Python I love, ditching it was probably the best decision for this project yet.
